src/domain/template/template/template.service.ts

```
@@ -71,77 +76,123 @@ export class TemplateService {
+          );
+        }
+        // Ensure no comments are created on the callout
+        templateData.calloutData.enableComments = false;
+        templateData.calloutData.visibility = CalloutVisibility.DRAFT;
+        templateData.calloutData.isTemplate = true;
+        templateData.calloutData.nameID = `template-${randomUUID().slice(0, 8)}`;
+        template.callout = await this.calloutService.createCallout(
+          templateData.calloutData!,
+          [],
+          storageAggregator
         );
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)